docs/mp/websocket/01_overview.adoc

```diff
@@ -42,7 +68,34 @@ CDI. Developers can choose between annotated and programmatic endpoints or use
 any combination of them. Using annotated endpoints is recommended in MP as
 they usually result in more succinct and easier-to-read code.
 
-== Helidon MP Example
+== API
+
+
+|===
+|Annotation |Description
+
+|@ServerEndpoint
+|This class level annotation declares that the class it decorates is a web socket endpoint that will be deployed and made available in the URI-space of a web socket server. The annotation allows the developer to define the URL (or URI template) which this endpoint will be published, and other important properties of the endpoint to the websocket runtime, such as the encoders it uses to send messages.
+
+|@ClientEndpoint
+|The ClientEndpoint annotation a class level annotation is used to denote that a POJO is a web socket client and can be deployed as such. Similar to ServerEndpoint, POJOs that are annotated with this annotation can have methods that, using the web socket method level annotations, are web socket lifecycle methods.
```

Comment:
"annotation a class level annotation is" -> "annotation, a class-level annotation, is"

Reply:
Fixed